### PR TITLE
Added a blacklist to CheckLinks

### PR DESCRIPTION
**Pull request by autonomousapps**
This is blocking https://github.com/gradle-guides/using-build-cache/pull/41

I haven't contributed code to this repo before, so I wasn't sure what conventions to follow, which is why I just added the new class as a nested static class. I also feel a little dirty not adding any tests, but I couldn't find any existing tests of this class.

As I said, this is blocking something elsewhere, so I really just wanted to get it in. Happy to have feedback on how to better structure this.

### CODE REVIEW

**Comment by autonomousapps** [comment]
cc @wolfs

**Comment by autonomousapps** [comment]
@eriwen once this change gets merged, we'll also have to bump the version and then publish that new version. Can you help with that tomorrow?

File: src/main/groovy/org/gradle/guides/CheckLinks.groovy [hunk 1]
```diff
@@ -78,11 +82,21 @@ class CheckLinks extends DefaultTask {
...
     }
 
     private Object indexDoc
-}
 
+    @CompileStatic
+    private static class Blacklist {
+
+        // These hosts are blocking web scrapers.
+        // https://github.com/gradle-guides/using-build-cache/pull/41#issuecomment-446375990
```
Comment (eskatos) [inline src/main/groovy/org/gradle/guides/CheckLinks.groovy]: :o: I would remove the link from this comment

Comment (autonomousapps) [reply src/main/groovy/org/gradle/guides/CheckLinks.groovy]: Sure!

**Pull request merged**